proto/attest.proto

```diff
@@ -65,11 +65,51 @@ message Event {
   bool digest_verified = 5;
 }
 
+message Certificate {
+  // Common, publicly-listed certificates by different vendors.
+  enum WellKnownCertificate {
+    UNKNOWN = 0;
+
+    // Microsoft certs: https://go.microsoft.com/fwlink/p/?linkid=321192
+    MS_WINDOWS_PROD_PCA_2011 = 1;
+    MS_THIRD_PARTY_UEFI_CA_2011 = 2;
+  }
```

Comment:
Done. Opted for the latter since the former generates an enum `pb.Certificate_WellKnown` and a struct for field 2 `pb.Certificate_WellKnown_`, which is confusing